Assets/Styles/bynder-editor.scss

```diff
@@ -1,18 +1,19 @@
 .bynder-trigger {
-    background-image: url('../images/bynder-logo.png');
+    background-image: url("../images/bynder-logo.png");
     background-size: contain;
     background-repeat: no-repeat;
-    padding-left: 2em !important;

```

Comment:
Have you checked if removing `!important` impacts this styling?

Reply:
This should be documented somewhere, that this may cause this styling to break, with some hints to use a more specific selector if it does break it.